### ISSUE TITLE
Choice of a custom voice for TTS

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#1151]
Some users have installed specific voices on their Windows system and would like to use them when using Thorium in TTS mode. But there is no UX allowing such a choice.

"I cannot change the narrator voice it uses to the one default on my system (Microsoft Hazel). The read out loud voice in Thorium is very grating and robotic. I would appreciate it if your program can utilize more TTS voices similar to the new Microsoft Edge browser and Adobe Reader."

"I have bought a couple of IVONA and Nuance voices (more natural sounding) These voices can be used directly in windows, however I cant change them to be used in your Thorium client. "  

This issue is different from #1130 but still related. The 1130 issue is about the language selection of the system voice vs the language indicated in the publication, and the possible language override (e.g. from fr-FR to fr-CA).

**Comment by danielweck** [comment]
Fixed via https://github.com/edrlab/thorium-reader/commit/ba01cb6bd74b2dadf37667bc1eb27c30f4cb13d1
(and https://github.com/readium/r2-navigator-js/commit/946265d4adb91126cd2937205453a756cb5b9962 in the `r2-navigato-js` component)

Please read some explanations here: https://github.com/edrlab/thorium-reader/issues/1130#issuecomment-817189211
